ipc-client/CMakeLists.txt

```diff
@@ -17,9 +17,9 @@ set_target_properties(herbstclient PROPERTIES
 
```

Comment:
Should the POSIC read POSIX a couple of lines above?

Reply:
Yes, I've fixed it in 26363fd5935d82dd2d3225f65ed05e40205113d7 and pushed it directly to master.

Reply:
Thanks!